scripts/config-build.py

```diff
@@ -98,6 +98,12 @@ def setup_ats(scriptsdir, buildpath):
                     action='store_true',
                     help="build third party libraries")
 
+#parser.add_argument("-gvz",
```

Comment:
Why not keep this?

Reply:
Because it is easier to just pass the `--graphviz` command in at the command line when running `config-build`.

Reply:
I actually added it back in with a unchangeable output name of `dependency.dot`, and then did the conversion to png automatically.